src/testdir/test_autocmd.vim

```diff
@@ -2002,51 +2002,101 @@ endfunc
 
 func Test_Cmdline_Trigger()
   autocmd CmdlineLeavePre : let g:log = "CmdlineLeavePre"
+  autocmd CmdlineLeavePre : let g:log2 = "CmdlineLeave"
```

Comment:
What is the point of having two almost identical autocommands?

Reply:
Fixed: https://github.com/vim/vim/pull/17217

There was an occasion where CmdlineLeave was triggered but CmdlineLeavePre was not.